src/utils/codemods/findAvailableCodemodsInNodeModules.js

```diff
@@ -2,32 +2,103 @@ const log = require('@dhis2/cli-helpers-engine').reporter
 const fs = require('fs-extra')
 const path = require('path')
 
-const isNotDotFolder = value => value !== '.' && value !== '..'
+const isDotFolder = name => name[0] === '.'
+
+const isNotDotFolder = name => !isDotFolder(name)
 
 const isDotJSFile = name => new RegExp('[.]js$').test(name)
 
-const extractCodemods = (paths, moduleName) => {
-    const { DHIS2_NODE_MODULES } = paths
-    const codemodsPath = path.join(DHIS2_NODE_MODULES, moduleName, 'codemods')
+const isFolder = path => {
+    if (!fs.existsSync(path)) return false
+    return fs.lstatSync(path).isDirectory()
+}
+
+const hasCodemods = codemodsPath =>
+    !!fs.readdirSync(codemodsPath).filter(curFile => {
+        const curPath = path.join(codemodsPath, curFile)
+
+        if (isFolder(curPath)) return false
+        if (!curFile.match(/\.(j|t)sx?$/)) return false
```

Comment:
(minor) this should probably use the `isDotJSFile` helper (and update the regex) right?

Reply:
Yeah, looks like it should - a `.tsx` file will pass here but fail later.  Also not sure what a `.jsx` or `.tsx` file would mean here.... also also not sure we support typescript?

Reply:
Ah, no idea how that happened. Updated